lib/traject/config/sirsi_config.rb

```diff
@@ -62,8 +62,11 @@
 
   provide 'solr_writer.commit_on_close', true
   provide 'mapping_rescue', (lambda do |context, e|
-    Honeybadger.notify(e, context: { record: context.record_inspect, index_step: context.index_step.inspect })
-
+    Honeybadger.notify(e, context: {
+      context: context.record_inspect,
+      record: begin context.source_record.to_honeybadger_context rescue nil end,
```

Comment:
Some records can be pretty big (like, a small record is 50KB, and any moderately complex serial is going to be 1MB+). Is there a subset of information (that might be easier to read over in honeybadger) that might be helpful? We're already getting the catkey and can always look up the whole record in FOLIO anyway.

Reply:
That's why I made this a method, which makes it easy to adjust what is returned.